src/main/java/frc/robot/Constants.java

```diff
@@ -29,6 +29,7 @@
   public static final int SPARKMAX_SHOOTER_CAN_ADDR_RIGHT = 5;
   public static final int SPARKMAX_SHOOTER_CAN_ADDR_LEFT = 6;
   public static final int SPARKMAX_LOADER_CAN_ADDR = 7;
+  public static final int SPARKMAX_SHOOTER_HOOD_CAN_ADDR = 8;
```

Comment:
The CAN addresses have not been selected for any of the SparkMAXs.  Sam is supposed to do this and up date the system chart at tonight's meeting.  Bug him until he gets this done.